web/src/js/components/Header/FlowMenu.tsx

```diff
@@ -222,6 +255,7 @@ const markers = {
 
 function MarkButton({ flow }: { flow: Flow }) {
     const dispatch = useAppDispatch();
+    const selectedFlows = useAppSelector((state) => state.flows.selected);
```

Comment:
Add a TODO comment here for multi-selection to indicate intent (it can/should be done, it just hasn't been done yet).